### DISCUSSION TITLE
Why are the latency results not displayed on the console

### DISCUSSION THREAD

**Discussion by Just-secularized-monk** [eclipse-zenoh/roadmap#128] [Q&A]
Hello, I have a question, I run z_pong and z_ping, why is the latency result not shown on the console?
<img width="612" alt="Why are the latency results not displayed on the console" src="https://github.com/eclipse-zenoh/roadmap/assets/37646573/df090261-c748-4c69-81bf-d7e55256d04f">

**Comment by YuanYuYuan** [comment]
Hi @Just-secularized-monk , can you try it again and ensure z_pong runs before z_ping? Furthermore, you can set `export RUST_LOG=info` to print out the debugging log.

**Reply by Just-secularized-monk** [reply]
Hello, I print out the debugging log, the attachment shows the 2 types of problems that exist, how to solve them? Thank you!
<img width="549" alt="debugging log" src="https://github.com/eclipse-zenoh/roadmap/assets/37646573/21e3648f-5ae5-48e4-b993-8f7046f27e38">

**Reply by YuanYuYuan** [reply]
It seems that they aren't connected. Can you check if the following commands work for you?

```bash
./z_pong -l tcp/127.0.0.1:7447
```
```bash
./z_ping -e tcp/127.0.0.1:7447 1024
```

**Reply by Just-secularized-monk** [reply]
Hello, the above commands don't work for me
<img width="545" alt="problem" src="https://github.com/eclipse-zenoh/roadmap/assets/37646573/afc75503-257a-4b36-bc3f-64e786b55caf">

**Reply by YuanYuYuan** [reply]
Hi @Just-secularized-monk. The examples seem not compiled with the default feature flags. Could you please run `cargo build` under the root of zenoh project and check if this works for you?

**Reply by gabrik** [reply]
Seems that you did not build with the `TCP` transport enabled, can you please run `cargo build --example z_ping --example z_pong` and try again?

**Reply by YuanYuYuan** [reply]
@gabrik I think the problem is @Just-secularized-monk may run the cargo under the examples directory instead of the root of project. See this [PR](https://github.com/eclipse-zenoh/zenoh/pull/854#issuecomment-2011362216) for details.

**Reply by Just-secularized-monk** [reply]
Yeah, you are right! I run the cargo under the root of project, and the result is shown on the console. Thank you!
